event_socket_server/base_server.py

```diff
@@ -16,6 +18,7 @@ def __init__(self, data, callback):
         self.callback = callback
 
 
+@total_ordering
```

Comment:
Did you read the documentation? You need `__eq__`. This is not C++ which emulates `a == b` with `!(a<b) && !(b<a)`.